web/pages/api/v0/question/[id]/index.ts

```diff
@@ -17,6 +17,6 @@ export default async function handler(
     res.status(404).json({ error: 'Contract not found' })
     return
   }
-  res.setHeader('Cache-Control', marketCacheStrategy)
-  return res.status(200).json(toFullMarket(contract))
+  res.setHeader('Cache-Control', questionCacheStrategy)
+  return res.status(200).json(toFullQuestion(contract))
```

Comment:
I guess this would break the api for everyone using `/market`. I'm not sure that's worth doing...
Can think about it